### ISSUE TITLE
Adapter has to have both Database and Resource

### ISSUE THREAD

**Issue by getmmg** [JonaszJestem/admin-bro-firebase#10]
Followed the example, but I am getting "Adapter has to have both Database and Resource" error. Any idea why this is happening?

**Comment by JonaszJestem** [comment]
Which version of `admin-bro` and `admin-bro-firebase` are you using?

**Comment by getmmg** [comment]
"admin-bro": "2.0.3",
"admin-bro-firebase": "^1.0.11",

**Comment by JonaszJestem** [comment]
Are you using `const AdminBroFirebase = require('admin-bro-firebase');` to import the module?
For quick workaround use:
`const AdminBroFirebase = require('admin-bro-firebase').default;`

**Comment by getmmg** [comment]
Yes I am using const AdminBroFirebase = require('admin-bro-firebase'); to import the module. Adding ,default to the import statement does fix it.

**Comment by onmarkets-development** [comment]
I have the same issue my project dose not work with it any update ?
